src/01-functions.js

```diff
@@ -74,7 +82,9 @@ function multiply() {}
  *  average(10, 6);
  *  //> 8 // 10 + 6 = 16; 16/2 = 8;
  */
-function average() {}
+function average(a, b) {
+  return ((a+b)/2);
```

Comment:
I would space out the operations happening inside your parenthesis on 86.  Also we dont need to wrap the entire operation in parenthesis as `a` and `b` will be added together first ( they're in parenthesis ) then the division will occur.